test/spanner.ts

```diff
@@ -5061,6 +5061,62 @@ describe('Spanner with mock server', () => {
       );
     });
   });
+
+  it('Plumb ObservabilityConfig into Spanner, Instance and Database', () => {
```

Comment:
Rename this test `Should use passed ObservabilityOptions in Spanner, Instance and Database"

Reply:
Done, thanks!